reducer.sh

```diff
@@ -2937,7 +2941,7 @@ verify(){
         else
           echo_out "$ATLEASTONCE [Stage $STAGE] Verify attempt #1: Maximum initial simplification & cleanup"
           grep -E --binary-files=text -v "^#|^$|DEBUG_SYNC|^\-\-| \[Note\] |====|  WARNING: |^Hope that|^Logging: |\++++| exit with exit status |Lost connection to | valgrind |Using [MSI]|Using dynamic|MySQL Version|\------|TIME \(ms\)$|Skipping ndb|Setting mysqld |Binaries are debug |Killing Possible Leftover|Removing Stale Files|Creating Directories|Installing Master Database|Servers started, |Try: yum|Missing separate debug|SOURCE|CURRENT_TEST|\[ERROR\]|with SSL|_root_|connect to MySQL|No such file|is deprecated at|just omit the defined" $WORKF \
-            | sed -e 's/;[\t ]*#.*/;/i' \
```

Comment:
This is done because in QC we have this:
`SELECT SUBSTRING('0', 2);#@25218413#NOERROR#WARNINGS: 0#CHANGED: 0#38`
and part `#@25218413` must stay because it's used later on, but this line removes anything after first `#` - not sure if it's important for anything else other then QC.

Reply:
Can we put a condition for QC? Otherwise, reduced testcase will have extra characters after semicolumn.